### ISSUE TITLE
Output of settings print can be unparsable under strange conditions.

### ISSUE THREAD

**Issue by scwoods1947** [OpenCollarTeam/OpenCollar#951]
What version of OpenCollar are you using? 8.2.300

What behavior did you expect? That the output of setting>print would work in a settings NC

What behavior did you see instead?  If you have more that 6 owners, the auth=owner~  uuid list ends up with the 7th uuid truncated and the remainder is places on a auth+owner~  which makes the whole bunch unparseable.

What steps does someone need to take to reproduce the problem?
Add 7 owners,  print settings and try to add the result auth=owner strings to a settings.
NC   unusable strings.txt](https://github.com/OpenCollarTeam/OpenCollar/files/11368046/OC.unusable.strings.txt)

**Comment by Medea-Destiny** [comment]
The idea here is that the auth+owner token gets added to the string generated from the auth=owner token, so the broken UUID is no longer broken after concatenation. Have you observed any issues caused here other than being a bit hard to read?

**Comment by scwoods1947** [comment]
Yes,  the list is not correct it's missing two people but has three lines
The viewer interprets the strings this is from chat.txt using notepad++
 Access List:
 Owners:
    secondlife:///app/agent/65c1c63d-8aaa-4a5c-8314-38d324efbb1a/about
    secondlife:///app/agent/ea0ccd44-894e-4b52-b564-6c86b652b29e/about
    secondlife:///app/agent/0e5dc170-4485-49c3-8578-6d9da2f742e2/about
    secondlife:///app/agent/179af4bc-4851-44cc-9c61-9fa08fe54df6/about
    secondlife:///app/agent/ba72a1e0-0a64-4841-9ed1-d8782f02ffb0/about
    secondlife:///app/agent/0c15ffa7-89a7-45ae-9a63-abb7be78c117/about
    secondlife:///app/agent/82836db8-7e41-4627-972/about
    secondlife:///app/agent/c-3e8001de6c2c/about

**Comment by scwoods1947** [comment]
Now on 8.3.0   the problem still persists.  It seems to me that given the information above (the list of app/agents above that the lines are NOT being properly concatenated.  I think that this is reasonable.    one, of several and in my opinion the simplest, proper solution would be to adjust the output format from coma separated items to line per uuid entries, this would:
 (1) avoid having input problems and complex input parsing.  and (2) Make the input clear to the end user.

**Comment by NikkiLacrima** [comment]
I just tested this with 8 owners  on an 8.3 collar. And sorry cannot reproduce the issue.

Did:
    settings print 
    copied the output exactly from local chat into the .settings notecard in inventory
    copied this .settings notcard into the collar
    runaway to clean out all owners
    settings load

No problems found with owners list.

The .settings had a line break in the middle of the 7th owner uuid with the final part on the next line beginning with "auth+owner~", this was parsed correctly by the settings load.

To reproduce/test we need to see the exact content of the .settings notecard that was used, and the exact steps use to transfer the output from "settings print" into .settings notecard.

**Comment by scwoods1947** [comment]
Well, I just tried it on an 8.3 collar, copying the entire output of settings print into a Notecard
Clearing the old .settings and saving it (collar rereads .setting on change)
runaway
checking that owner list is empty
Pasting the contents of the notecard into .settings and saving it. (collar rereads .settings again)
checking the ownerlist, which is correct.
So  I stand corrected.   Thank you
Lets close this.
